include/tsl/ordered_hash.h

```diff
@@ -1098,6 +1098,44 @@ class ordered_hash : private Hash, private KeyEqual {
     return 1;
   }
 
+  /**
+   * Clear a bucket without touching the container holding the values.
+   */
+  void clear_bucket(typename buckets_container_type::iterator it) {
```

Comment:
The clear_bucket should be a private method, or more simply a const lambda in erase_if.

Reply:
I made it a lambda.